pkg/operator/ceph/file/filesystem_test.go

```diff
@@ -327,6 +400,42 @@ func TestCreateFilesystem(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+func TestUpgradeFilesystem(t *testing.T) {
```

Comment:
Please add at least one error case. For example, what if one of the MDSes can't be stopped?